ref: update `exp init` intro. (#3440)
iterative/dvc.org: content/docs/command-reference/exp/init.md

## Description

`dvc exp init` helps you get started with DVC Experiments quickly.

> It creates an experiment pipeline in a dvc.yaml file, which assumes standard locations for the inputs (source code, data, and parameters) and outputs (model, metrics, and plots) of your model training command.

Author: ```suggestion
It creates an experiment pipeline in a dvc.yaml file, which assumes standard locations for the inputs (source code, data, and <abbr>parameters</abbr>) and outputs (model, <abbr>metrics</abbr>, and <abbr>plots</abbr>) of your model training command.
```

## Description

`dvc exp init` helps you get started with DVC Experiments quickly. It prepares

> an experiment in a `dvc.yaml` file, which assumes standard locations for the

Author: ```suggestion
an experiment in `dvc.yaml`, which assumes standard locations for the
```

> `dvc exp init` helps you get started with DVC Experiments quickly. It creates an
> experiment pipeline in a dvc.yaml file, which assumes standard locations for the

Author: @dberenbaum I don't think we should use the pipeline concept here, WDYT?
```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It creates an
experiment inside a `dvc.yaml` file, which assumes standard locations for the
```

Reviewer: Also need backticks around `dvc.yaml` whatever we decide.

Author: > dvc exp init doesn't create an experiment

It "creates" it in the sense that it codifies it but yeah since it's not run yet maybe that wasn't the best wording... 🤔

Author: Committing this for now:

```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It prepares an
experiment in a `dvc.yaml` file, which assumes standard locations for the
```

inputs (source code, data, and <abbr>parameters</abbr>) and outputs (model,

> <abbr>metrics</abbr>, and <abbr>plots</abbr>) of your model training command.

Author: @daavoo feel free to add the metrics (and plots if you want) tooltip files in this PR (if you have time soon, otherwise no worries).

> These locations can be customized through the [options](#options) below or via
> [configuration](/doc/command-reference/config#exp).
> 
> Repository structure assumed by default:

Author: ```suggestion

The locations can be customized through the [command options](#options) or via
[configuration](/doc/command-reference/config#exp). Default structure:
```

## Description

> `dvc exp init` helps you get started with DVC Experiments quickly. It prepares
> an experiment in `dvc.yaml`, which assumes standard locations for the inputs

Reviewer: ```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It prepares
the experiment structure in `dvc.yaml`, which assumes standard locations for the inputs
```

Feel free to reword, but I think "prepares an experiment" makes it sound like it's something you do for every experiment instead of a framework for all your experiments.

Reviewer: Actually, I think it's hard to separate this from #3430. These look they will inevitably conflict.

Author: Overlap is OK, we can solve conflicts 🙂

Author: Hmmm wait yeah you're right. This is now included completely in #3430... Closing this PR then.
